liblangutil/Scanner.cpp

```diff
@@ -76,7 +76,7 @@ string to_string(ScannerError _errorCode)
 		case ScannerError::IllegalHexDigit: return "Hexadecimal digit missing or invalid.";
 		case ScannerError::IllegalCommentTerminator: return "Expected multi-line comment-terminator.";
 		case ScannerError::IllegalEscapeSequence: return "Invalid escape sequence.";
-		case ScannerError::IllegalCharacterInString: return "Invalid character in string.";
+		case ScannerError::IllegalCharacterInString: return "Invalid character in string. Unicode character in non-unicode string literal; literal must be marked as unicode";
```

Comment:
This is used for any non-printable ascii character, so we cannot know that the user intended this to be unicode. Maybe something like `"If you want to use unicode characters, use a unicode\"...\" string literal."`